rfcs/spinner-component.md

```diff
@@ -0,0 +1,30 @@
+# Spinner
+
+## Usage
+
+### Basic usage
+
+```javascript
+import * as React from 'react';
+import {Spinner} from 'baseui/spinner';
+
+export default () => {
+  return
+    <Spinner
+      variant={Spinner.VARIANT.INDETERMINATE}
+      size={10}
+    />;
+};
+```
+
+## Exports
+
+* `Spinner`
+* `VARIANT`
+
+## `Spinner` API
```

Comment:
`size` appears to be missing below though its in the usage example above.

In addition to size I think we probably want some story around customizing the color. This could probably be done via `overrides`, but it seems like overriding the color may be a common enough use case that supporting a `color: string` prop may be worth it (superfine had something similar).

```js
<Spinner size={10} color="white" />
```